src/KeyManager.ts

```diff
@@ -36,6 +36,7 @@ export interface SignTransactionParams {
   transaction: Transaction;
   id: string;
   password: string;
+  custom?: any;
```

Comment:
I would require this to be an object; if someone needs a bool or a string or something, they can make that a value in the object.